issue_template.md

```diff
@@ -0,0 +1,19 @@
+## Issue description
+
+If you found a bug or want to propose a feature, please describe it as accurately as possible. Use the following checklists. This way we can help you the best. 
+
+### Your checklist for a bug report
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the issue. If you're unsure about any of them, don't hesitate to ask. We're here to help!_
+
+- [ ] Have you described how to reproduce your bug?
+- [ ] Which operating system, browser and versions do you use?
+- [ ] Can you attach some logs or screenshots to this issue?
```

Comment:
I find check lists with questions irritating. Would it make sense to turn these into demands? Something like

#### Describe how to reproduce the bug
_Please provide a detailled description of what you have to do to reproduce your bug (settings, click paths, ...)_

#### Describe your SCM-Manager setup
_What plugins and what versions do you use? Are you using a server like tomcat? Do you have special settings like proxies? Are you running your instance behind a reverse proxy?_

#### Which operating system, browser and versions do you use?
_Windows, Max or Linux? Chrome, Firefox or IE? What versions?_

#### Log of the bug
_Please mind to check for any sensible information like repository names!_

#### Further information like screenshots